.github/workflows/test.yml

```diff
@@ -48,12 +46,7 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          if go help mod > /dev/null 2>&1 ; then
-            go mod download
-          else
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-          fi
+          go mod download
```

Comment:
`go mod download` tends to download much more than needed, especially if not actively `go mod tidy`ing the mod files. It's fine to remove this and let Go automatically download what's needed

Reply:
Great sounds good. Removed!